packages/test/test-end-to-end-tests/src/test/leader.spec.ts

```diff
@@ -11,7 +11,7 @@ import { describeFullCompat } from "@fluidframework/test-version-utils";
 
 async function ensureConnected(container: Container) {
     if (!container.connected) {
-        await timeoutPromise((resolve, rejected) => container.on("connected", resolve));
+        await new Promise((resolve, rejected) => container.on("connected", resolve));
```

Comment:
can we stick with timeoutPromise and just bump the timeout? in vscode we run without timeout on the test, so this can lead to a hang